examples/cactus-example-supply-chain-backend/src/main/typescript/infrastructure/supply-chain-app-dummy-infrastructure.ts

```diff
@@ -69,18 +68,18 @@ export class SupplyChainAppDummyInfrastructure {
   public static readonly CLASS_NAME = "SupplyChainAppDummyInfrastructure";
 
   public readonly besu: BesuTestLedger;
-  public readonly quorum: QuorumTestLedger;
+  public readonly xdai: BesuTestLedger;
```

Comment:
Can you call it `besuXdai`/`xdaiBesu` or `besu`? It may be confusing for someone not familiar with xdai to see it as besu ledger (at least I was confused at first :))

Reply:
@outSH Fair point and thank you for the suggestion, I changed it! :-)

Reply:
Great, thanks!